### ISSUE TITLE
Spreadsheet security

### ISSUE THREAD

**Issue by roblingle** [opensavannah/savmaps#3]
I think we should limit access to people who are logged into google at least. Ideally, people would request access and then be allowed in. Is there good reason to allow anonymous changes?

![v2_leaflet_maps_with_google_sheets_template_v1_1_1_-_google_sheets](https://user-images.githubusercontent.com/12083/29256521-05fd0eca-8079-11e7-9983-6cab5b765fc8.png)

**Comment by carlvlewis** [comment]
Yes, there's a very good reason for allowing open changes: so that *anyone* can go in and contribute and see their work in real time.. But for this project, unlike SavMaps, the level of technical expertise needed to format the new entries properly is most likely above the skillset of the average user. That said, the participants, who learned the process at the event, said they were eager to go home and add new data as they think of it.

**Comment by carlvlewis** [comment]
We'll set it to anonymous viewers allowed but no anonymous edits. How about that?

There's also always revision history...

**Comment by roblingle** [comment]
Anonymous viewing sounds great.

We can find approaches to keep it easy to contribute while still requiring a login. Reverting gets messy quickly when there are some good and some bad edits. Putting people's names by their changes gives them some ownership, personal pride in, and responsibility for the result.

The README top line can include an email address- `yall@` or similar- that people should reach out to for access.
